scripts/plot_network.py

```diff
@@ -259,25 +258,31 @@ def split_costs(n):
 
     set_plot_style()
 
-    paths, config, wildcards, logs, out = retrieve_snakemake_keys(snakemake)
+    paths, config, wildcards, logs, out = (
+        snakemake.input,
+        snakemake.config,
+        snakemake.wildcards,
+        snakemake.log,
+        snakemake.output,
+    )
```

Comment:
Technically, this is correct, but in some prior discussions we've decided to not use these names/abbreviatons anymore. Could you maybe therefore use directly `snakemake.input` instead of `paths` and `snakemake.output` instead of `out`? The logs are not needed, as far as I can see. We could keep the remaining abbreviations 
```suggestion
    config, wildcards = snakemake.config, snakemake.wildcards
```